backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -34,7 +34,7 @@ def index
           params[:q][:completed_at_lt] = params[:q].delete(:created_at_lt)
         end
 
-        @search = Order.preload(:user).accessible_by(current_ability, :index).ransack(params[:q])
+        @search = Order.preload(:user).accessible_by(current_ability, :index).ransack(params[:q].to_unsafe_h)
```

Comment:
@priyank-gupta @mauazua I think that it's ok for now (untill ransack will have it's own solution), because:
1. It's admin, and in admin permitted params aren't used even for editing products.
2. Ransack never used permitted params, only whitelisting.